File: javascript/example_refactoring.js
// Before refactoring:
// function getUserInfo(user) { return user.name + ', ' + user.age + ' years old'; }

// After refactoring:

function formatUserInfo(user) {
    return `${user.name}, ${user.age} years old`;
}

// Usage:
// console.log(formatUserInfo(someUser));

File: go/example_refactoring.go
// Before refactoring:
// func getUserInfo(user User) string { return user.Name + ", " + user.Age + " years old" }

// After refactoring:

func formatUserInfo(user User) string {
    return user.Name + ", " + user.Age + " years old"
}

// Usage:
// fmt.Println(formatUserInfo(someUser))
